sulong/mx.sulong/suite.py

```diff
@@ -711,10 +711,12 @@
         "LIBCXX_CXX_ABI_INCLUDE_PATHS" : "<path:sdk:LLVM_ORG_SRC>/libcxxabi/include",
         "LIBCXX_ENABLE_STATIC" : "NO",
         "LIBCXX_ENABLE_EXPERIMENTAL_LIBRARY" : "NO",
+        # '{{musl}}' is replaced with YES or NO in the CMakeProject.
+        "LIBCXX_HAS_MUSL_LIBC" : "{{musl}}",
```

Comment:
I think this should use the `os_arch` mechanism of mx for this (see also current master, where we already use this for windows specific stuff):
```
"os": {
   "linux-musl": {
      "cmakeConfig": {
         "LIBCXX_HAS_MUSL_LIBC": "YES",
      }
   }
}
```

Reply:
Looks much better indeed.